pkgs/development/tools/kube-prompt/default.nix

```diff
@@ -11,7 +11,7 @@ buildGoPackage rec {
     inherit rev;
     owner = "c-bata";
     repo = "kube-prompt";
-    sha256 = "09c2kjsk8cl7qgxbr1s7qd9br5shf7gccxvbf7nyi6wjiass9yg5";
+    sha256 = "1c1y0n1yxcaxvhlsj7b0wvhi934b5g0s1mi46hh5amb9j3dhgq1c";
   };
 
   goDeps = ./deps.nix;
```

Comment:
I think this needs to be regenerated since the `Gopkg.lock` changed during this release https://github.com/c-bata/kube-prompt/commit/eb06ef3efdc5ffa63696a944618b5df2a69be2c2

Reply:
@worldofpeace oh you are right, my bad :sweat_smile: I'll update tomorrow morning :wink: